test-e2e/manager-basic.js

```diff
@@ -56,20 +59,90 @@ test('Managing created projects', async (t) => {
   t.ok(project1)
   t.ok(project2)
 
-  t.test('initial settings from project instances', async (st) => {
-    const settings1 = await project1.$getProjectSettings()
-    const settings2 = await project2.$getProjectSettings()
-
-    st.alike(settings1, {
-      name: undefined,
-      defaultPresets: undefined,
-    })
+  await t.test(
+    'initial settings and default config from project instances',
+    async (st) => {
+      const settings1 = await project1.$getProjectSettings()
+      const settings2 = await project2.$getProjectSettings()
+
+      const {
+        presets: defaultPresets,
+        fields: defaultFields,
+        icons: defaultIcons,
+      } = await getExpectedConfig(
+        new URL('../config/defaultConfig.mapeoconfig', import.meta.url).pathname
+      )
+
+      st.is(settings1.name, undefined, 'undefined name for project1')
+      st.is(settings2.name, 'project 2', 'matched name for project2')
+
+      const projectPresets = await project1.preset.getMany()
+
+      st.is(
+        settings1.defaultPresets?.point.length,
```

Comment:
Better to `@ts-ignore` here and let it throw, rather than using `?`, since it could obscure a bug in the test.